pkg/api/types.go

```diff
@@ -415,3 +425,33 @@ func (il *InjectionList) String() string {
 func (il *InjectionList) Type() string {
 	return "string"
 }
+
+// Set implements the Set() function of pflags.Value interface.
+func (e *EnvironmentList) Set(value string) error {
+	parts := strings.SplitN(value, "=", 2)
+	if len(parts) != 2 || len(parts[0]) == 0 {
+		return fmt.Errorf("invalid environment format %q, must be NAME=VALUE", value)
+	}
```

Comment:
doesn't this prevent me from having an env variable who's value includes an = sign?

MYVAR=FOO=2

where i want "FOO=2" to be the value?